articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Azure AD v2 JavaScript SPA quickstart | Microsoft Docs
+description: Learn how JavaScript SPA applications can call an API that require access tokens by Azure Active Directory v2.0 endpoint
+services: active-directory
+documentationcenter: dev-center-name
+author: andretms
+manager: mtillman
+editor: ''
+
+ms.service: active-directory
+ms.component: develop
+ms.devlang: na
+ms.topic: article
+ms.tgt_pltfrm: na
+ms.workload: identity
+ms.date: 07/10/2018
+ms.author: andret
+ms.custom: aaddev
+
+#Customer intent: As an app developer, I want to learn how to get access tokens using the Azure AD v2.0 endpoint so that my JavaScript app can sign in personal and work and school accounts.
+---
+
+# Sign in users and acquire an access token from a JavaScript application
+
+In this quickstart, you'll learn how to use a code sample that demonstrates how a JavaScript single page application (SPA) can sign in personal accounts, work and school accounts, get an access token, and call the Microsoft Graph API.
+
+![How the sample app generated by this Quickstart works](media/active-directory-javascriptspa/javascriptspa-intro.png)
+
+> [!div renderon="docs"]
+> ## Register your application and download your quickstart app
+> You have two options to start your quickstart application:
+> * [Express] [Option 1: Register and then download your quickstart application](#option-1-register-your-application-and-download-your-quickstart-app)
+> * [Manual] [Option 2: Register your application and configure the sample](#option-2-register-and-manually-configure-your-application-and-code-sample)
+>
+> ### Option 1: Register your application and download your quickstart app
+>
+> 1. Go to the [Azure portal - Application Registration [Prod]](https://portal.azure.com/signin/index/?Microsoft_AAD_RegisteredApps=true#blade/Microsoft_AAD_RegisteredApps/applicationsListBlade/quickStartType/JavascriptSpaQuickstartPage/sourceType/docs).
+> 1. Enter a name for your application and click **Register**.
+> 1. Follow the instructions to download and automatically configure your new application for you in one click.
+> [!div renderon="docs"]
+> ### Option 2: Register and manually configure your application and code sample
+> #### Step 1: Register your application
+> 
+> 1. To register an application, go to the [Azure portal - Application Registration [Prod]](https://aka.ms/registeredappsprod) and select **New registration**.
+> 1. Enter a name for your application, add `http<span/>://localhost:30662/` in **Reply URL**, and click **Register**.
```

Comment:
There is a span tag in the url here by error.

Reply:
@navyasric - this is weird by was added on purpose as workaround - because otherwise the markdown converter would automatically translate this to an Hyperlink and we don't want this. I've actually learned that this auto-conversion does not happen if the hyperlink is in between `` - so I am removing this occurrence ;)

Reply:
Oh that's interesting. Didn't know that.